### ISSUE TITLE
LDC AST view reports no output

### ISSUE THREAD

**Issue by RabsRincon** [mattgodbolt/compiler-explorer#881]
The AST pane reports `<No output>` for LDC as seen https://godbolt.org/g/SQRkEb

cc @JohanEngelen

**Comment by JohanEngelen** [comment]
Things work locally. Could this be because LDC outputs the AST file in the same directory (!) as the source file?  (cannot be specified currently, possibly need to change compiler behavior upstream)

**Comment by RabsRincon** [comment]
I think I might have the solution. But first I need to know if the file is named `example.d.cg` or simply `example.cg`?

**Comment by RabsRincon** [comment]
As I suspected
```
Mar 24 02:16:56 06b5a890b22d unified: warn: LDC AST file /tmp/compiler-explorer-compiler118224-56-1dx4lsj.ohjr/example.d.cg requested but it does not exist 
Mar 24 02:16:56 06b5a890b22d unified: info:  - - [24/Mar/2018:01:16:56 +0000] "POST /api/compiler/ldc180/compile HTTP/1.0" 200 - "-" "Amazon CloudFront" 
```
 can be seen in the beta logs, meaning that this check https://github.com/mattgodbolt/compiler-explorer/blob/master/lib/compilers/ldc.js#L80 is failing, but afaik, it should be working! So weird!

**Comment by mattgodbolt** [comment]
Was a bug in the execution code! Fixed in f9fd86fc248ea3dc95bb440803fd41d990d3db12, sorry for all the confusion!
